src/telemetry/Manager.cc

```diff
@@ -135,6 +135,13 @@ void Manager::InitPostScript() {
                               []() { return static_cast<double>(get_stats()->fds); });
 #endif
 
+    // These two metrics get set at startup and are never modified after.
+    process_start_time = GaugeInstance("process", "start_time", {}, "Process start time", "seconds");
+    process_start_time->Set(std::trunc(util::current_time(true)));
```

Comment:
We have run_state::zeek_start_time, populated during setup() - how about using that?
```suggestion
    process_start_time->Set(run_state::zeek_start_time);
```


FWIW: uptime can be computed roughly be using `zeek_net_timestamp_seconds - process_start_time_seconds`.